Gemfile

```diff
@@ -38,6 +38,9 @@ gem 'omniauth'
 gem 'request_store', "~> 1.1.0"
 gem 'gravatar_image_tag', '~> 1.2.0'
 
+gem 'warden', '~> 1.2'
```

Comment:
let's change that to `~> 1.2.3` so we follow the semantic versioning paradigm more closely